src/nl/rubensten/texifyidea/run/compiler/BibtexCompiler.kt

```diff
@@ -18,10 +18,7 @@ internal object BibtexCompiler : Compiler<BibtexRunConfiguration> {
         val moduleRoots = ProjectRootManager.getInstance(project).contentSourceRoots
 
         command.apply {
-            if (runConfig.compilerPath != null) {
-                add(runConfig.compilerPath!!)
-            }
-            else add(executableName)
+            add(runConfig.compilerPath ?: BiberCompiler.executableName)
```

Comment:
@RubenSchellekens Running the `BiberCompiler.executableName` in a `BibtexCompiler` is not going to make sense, obviously. I think we need to run some manual tests for common usage before a release, we would have caught this bug.